plugins/hls-refactor-plugin/src/Development/IDE/GHC/ExactPrint.hs

```diff
@@ -623,6 +653,14 @@ eqSrcSpanA l r = leftmost_smallest l r == EQ
 #endif
 
 #if MIN_VERSION_ghc(9,2,0)
+
+spanContainsRange :: SrcSpan -> Range -> Bool
+spanContainsRange srcSpan Range {..} =
+  srcSpan `spans` positionToTuple _start && srcSpan `spans` positionToTuple _end
+  where
+    positionToTuple :: Position -> (Int, Int)
+    positionToTuple (Position l c) = (fromIntegral l + 1, fromIntegral c)
+
```

Comment:
Searching for this type signature (flipped) within the codebase, I find 3 functions that seem to duplicate this logic.

Worth reusing one of them? 

<img width="547" alt="Screenshot 2022-09-18 at 16 26 18" src="https://user-images.githubusercontent.com/26626/190912055-f26f913e-bca7-4a2c-803d-5fbc9bbc9fa5.png">

Reply:
I pulled this out into Development.IDE.GHC.Error, where similar functions are.